packages/p2p/src/stores/order-store.js

```diff
@@ -64,8 +71,46 @@ export default class OrderStore {
             p2p_order_confirm: 1,
             id,
         }).then(response => {
-            if (response && response.error) {
-                order_details_store.setErrorMessage(response.error.message);
+            if (response) {
+                if (response.error) {
+                    if (response.error.code === 'OrderEmailVerificationRequired') {
+                        clearTimeout(wait);
+                        const wait = setTimeout(() => this.setIsEmailVerificationModalOpen(true), 250);
+                    } else if (
+                        response?.error.code === 'InvalidVerificationToken' ||
+                        response?.error.code === 'ExcessiveVerificationRequests'
+                    ) {
+                        clearTimeout(wait);
+                        this.setVerificationLinkErrorMessage(response.error.message);
+                        const wait = setTimeout(() => this.setIsInvalidVerificationLinkModalOpen(true), 230);
```

Comment:
why not use 250 for timeout same as above?

Reply:
The last 2 conditions are fulfilled when the user comes back to p2p via a link and those needed lesser time to fulfill, I tried changing it to see what works best and it was `230` 😛